modules/database-commons/build.gradle

```diff
@@ -0,0 +1,5 @@
+description = "Testcontainers :: Database-Commons"
+
+dependencies {
+    compile project(':testcontainers')
```

Comment:
Should we maybe add this dependency for all subprojects but `:testcontainers`? I think simple would not need any `build.gradle` anymore in this case.

Reply:
I'm not a fan of implicit dependencies, to be honest :)
Plus IMO `description` have to be set by a human.

5 lines file per module shouldn't hurt too much :)